tools/SetupFlow/DevHome.SetupFlow/Services/QuickStartProjectService.cs

```diff
@@ -33,7 +33,7 @@ public async Task<List<QuickStartProjectProvider>> GetQuickStartProjectProviders
         {
             try
             {
-                var quickStartProjectProviders = (await extension.GetListOfProvidersAsync<IQuickStartProjectProvider>()).ToList();
+                var quickStartProjectProviders = (await extension.GetListOfProvidersAsync<IQuickStartProjectProvider2>()).ToList();

```

Comment:
What if this releases but no extensions implement `IQuickStartPRojectProvider2`?  `GetListOfProvidersAsync` will return 0 results.  This is true even if some providers implement `IQuickStartProjectProvider`